### PR TITLE
Improve subheadings/flow on organization/strategic-council

### PR DESCRIPTION
**Pull request by ElenaFdR**
Small restructuring plus clarification based on user feedback.

-----
[View rendered organization/strategic-council.md](https://github.com/publiccodenet/about/blob/improve-clarity-strategic-council/organization/strategic-council.md)

### CODE REVIEW

File: organization/strategic-council.md [hunk 1]
```diff
@@ -20,16 +20,24 @@ Members of the strategic council are expected to:
...
-This commitment is estimated to be an average of 15 minutes per week.
 This is a voluntary role and non-remunerated.
-Membership of the strategic council can be terminated at any time in writing by either the member of the strategic council or the board of directors.
 
 ## Appointments
 
 These are the steps for appointing a new strategic council member.
 
-1. Potential new strategic council members are either invited to apply by the Foundation for Public Code, or may nominate themselves.
+1. Potential new strategic council members are either invited to apply by the Foundation for Public Code, or may nosminate themselves.
```
Comment (Ainali) [inline organization/strategic-council.md:29]: This is probably a mistake.

Comment (ElenaFdR) [reply organization/strategic-council.md]: Good spot, thank you!

Comment (lhawthorn) [inline organization/strategic-council.md]: I believe that nominate is now misspelled. It does not contain an S. Apologies if I am a poor reader of diff files.

Comment (ElenaFdR) [reply organization/strategic-council.md]: I believe this is now fixed! Thanks for checking.

**Review by AlbaRoza — APPROVED** [review]
Good to go

**Review by ericherman — APPROVED** [review]

**Pull request merged**